### DISCUSSION TITLE
Batch download fails to start

### DISCUSSION THREAD

**Discussion by GODK1NG** [subhra74/xdm#652] [General]
Batch download fails to start after pressing OK on the "Batch download" pop-up

Can successfully right-click browser button and select "Download all links" to make "Batch download" window pop up, and select the various files to download, but when pressing OK, nothing happens and window persists

Hoping this will be fixed in the next update

**Comment by subhra74** [comment]
Will be fixed in next update, also would recommend to open a issue instead of discussion
